dashboard/src/actions/namespace.ts

```diff
@@ -17,7 +17,11 @@ export const errorNamespaces = createAction("ERROR_NAMESPACES", resolve => {
   return (err: Error, op: "list") => resolve({ err, op });
 });
 
-const allActions = [setNamespace, receiveNamespaces, errorNamespaces];
+export const clearNamespaces = createAction("CLEAR_NAMESPACES", resolve => {
```

Comment:
You can do `export const clearNamespaces = createAction("CLEAR_NAMESPACES")` if you don't need any payload.

Reply:
Thanks.